### ISSUE TITLE
User Export CSV broken due to missing linefeed / line break filter

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#4097]
**Description**: The user export is broken, if text values contain line breaks, since they aren't filtered

Environment: Shopware 6 backend exporter

**Steps to reproduce:**
1. Have a text value like street in customer_adress contain a line break (Check with MySQL like: 'SELECT * FROM `customer_address` WHERE street LIKE CONCAT('%', CHAR(10), '%')';  ((or 'CHAR(13)') This can happen for many reasons like for example if users were migrated from  other sources, or third party plugins like AcrisStreet were used
2. Start a user export with a profile containing this value(s).
3. Opening this CSV export into an excel table will assert many values to the wrong columns and create more lines than actual users

**Expected result: **
CSV line feeds only after each user entity
e.g.:
;"Max Zweitname";Mustermann;"Beispielweg 1";"12345";"Beispielort"
;Testalia;Test;"Testweg 2";"6789";"Testing"

**Actual result: **
CSV line feeds after each user entity AND in the middle of an entity where the value contained one
e.g.:
;"Max
Zweitname";Mustermann;"Beispielweg
1";"12345";"Beispielort"
;Testalia;Test;"Testweg
2";"6789";"Testing"

-> mismatched values and assumingly 5 instead of 2 users in excel

**Comment by shopwareBot** [comment]
[public] Automated response: This issue is linked to the internal issue [https://shopware.atlassian.net/browse/NEXT-31555](https://shopware.atlassian.net/browse/NEXT-31555).

 

[created from NEXT-31555, comment 485502]

**Comment by MalteJanz** [comment]
Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.
If you feel like this issue is still impacting you, please create a new issue and let us know. Thank you again for your valuable feedback.
